google/cloud/spanner_dbapi/parser.py

```diff
@@ -259,25 +255,4 @@ def expect(word, token):
         word = word[1:]
         return word, a_args(terms)
 
-    elif token == EXPR:
-        if word == "%s":
-            # Terminal symbol.
-            return "", pyfmt_str
-
-        # Otherwise we expect a function.
-        return expect(word, FUNC)
```

Comment:
Didn't find where `EXPR` is passed into the method, so I guess it's not actually used